### PR TITLE
[1.1] Update buildtools and cli version to same ones as CoreFX and CoreCLR

### PR DESCRIPTION
**Pull request by janvorli**
Having separate CLI version greatly increases the cost of adding new
distros, so we should keep coreclr, corefx and core-setup using the
same CLI and buildtools version for their tools.

It also unifies the init-tools.sh / cmd with the other repos.

### CODE REVIEW

File: pkg/Tools-Override/_ [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (weshaggard) [inline pkg/Tools-Override/_]: we should just remove the tools-override directory and part of the script.

File: pkg/init-tools.cmd [hunk 2]
```diff
@@ -65,10 +65,20 @@ echo Running: "%BUILD_TOOLS_PATH%init-tools.cmd" "%~dp0" "%DOTNET_CMD%" "%TOOLRU
...
 if not [%INIT_TOOLS_ERRORLEVEL%]==[0] (
-	echo ERROR: An error occured when trying to initialize the tools. Please check '%INIT_TOOLS_LOG%' for more details.
-	exit /b %INIT_TOOLS_ERRORLEVEL%
+  echo ERROR: An error occured when trying to initialize the tools. 1>&2
+  goto :error
 )
 
-:: Create sempahore file
+:: Always copy over the Tools-Override
+xcopy /vy "%~dp0Tools-Override\"* "%~dp0Tools" 1>nul
```
Comment (weshaggard) [inline pkg/init-tools.cmd]: Just remove this part.

File: pkg/init-tools.sh [hunk 3]
```diff
@@ -105,18 +117,29 @@ if [ ! -e $__INIT_TOOLS_DONE_MARKER ]; then
...
     echo "Running: $__BUILD_TOOLS_PATH/init-tools.sh $__scriptpath $__DOTNET_CMD $__TOOLRUNTIME_DIR" >> $__init_tools_log
     $__BUILD_TOOLS_PATH/init-tools.sh $__scriptpath $__DOTNET_CMD $__TOOLRUNTIME_DIR >> $__init_tools_log
     if [ "$?" != "0" ]; then
-        echo "ERROR: An error occured when trying to initialize the tools. Please check '$__init_tools_log' for more details."1>&2
+        echo "ERROR: Could not restore build tools correctly." 1>&2
+        display_error_message
         exit 1
     fi
+
+    cp Tools-Override/* Tools/
```
Comment (weshaggard) [inline pkg/init-tools.sh]: Just remove this part.

File: pkg/Tools-Override/_ (hunk 1 above)
Comment (janvorli) [reply pkg/Tools-Override/_]: Even though that makes it different from the corefx version?

Comment (weshaggard) [reply pkg/Tools-Override/_]: It just feels pointless to add this empty file. You could also add a exists check in the scripts perhaps.

Comment (weshaggard) [reply pkg/Tools-Override/_]: I don't have a strong opinion but I could definitely see someone looking at this in the future and asking what it is for and even try deleting it.

Comment (janvorli) [reply pkg/Tools-Override/_]: Ok, the exists check sounds good. I'll do that.

**Comment by janvorli** [comment]
@weshaggard I've made the folder optional.

**Comment by rakeshsinghranchi** [comment]
Thanks @janvorli !

**Pull request merged**